src/Chip.js

```diff
@@ -38,10 +39,11 @@ const Chip = ({
             disabled,
             dragging,
         })}
+        data-test={dataTest}
     >
         <Icon icon={icon} />
         <Content overflow={overflow}>{children}</Content>
-        <Remove onRemove={onRemove} />
+        <Remove onRemove={onRemove} dataTest={`${dataTest}-remove`} />
```

Comment:
Not sure if this is necessary. If we gave the components in the sub folders a default value as well, the `Remove` would have `dhis2-uicore-chip-remove` and could be targeted like this: `[data-test="custom-chip-id"] [data-test="dhis2-uicore-chip-remove"]`.

This would eliminate the need to create these dynamic attributes and simplify our components' code a bit by not having to add the dataTest unless we explicitly need it

Reply:
We talked a bit about this on Slack, in the end we decided to stick with the existing approach.